modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-multiple-ingress-through-single-alb_{context}"]
+= Creating multiple ingresses through single ALB
+
+You can route the traffic to multiple Ingresses through a single AWS Load Balancer (ALB).
+
+.Procedure
+
+. Create an `IngressClassParams` resource:
++
+[source,yaml]
+----
+apiVersion: elbv2.k8s.aws/v1beta1 <1>
+kind: IngressClassParams
+metadata:
+  name: single-lb-params <2>
+spec:
+  group:
+    name: single-lb <3>
+----
+
+<1> Defines the API group and version of the `IngressClassParams` resource.
+<2> Specifies the name of the `IngressClassParams` resource.
+<3> Specifies the name of the `IngressGroup`. All Ingresses of this class belong to this `IngressGroup`.
+
```

Comment:
Assuming that this is being done from the CLI, do we need to add a step for clarifying how to create the resource?For example, by running some oc command?

Reply:
Addressed- updated the `oc create` command for all the instances